components/mpas-ocean/src/analysis_members/Registry_conservation_check.xml

```diff
@@ -198,6 +198,53 @@
 			 description="Salinity flux from frazil to Land Ice, given to coupler. Positive into the ocean."
 		/>
 	</var_struct>
+            <var_struct name="conservationCheckCarbonAM" time_levs="1" packages="conservationCheckAMPKG">
+                <var name="initialCarbon" type="real" dimensions="Time" units="kg"
+                     description="Total ocean carbon at start of conservation interval"
+                />
+                <var name="finalCarbon" type="real" dimensions="Time" units="kg"
+                     description="Total ocean carbon at end of conservation interval"
+                />
+                <var name="carbonChange" type="real" dimensions="Time" units="kg"
+                     description="Change in ocean carbon over conservation interval"
+                />
+                <var name="netCarbonFlux" type="real" dimensions="Time" units="kg"
+                     description="Net surface flux of ocean carbon over conservation interval"
+                />
+                <var name="absoluteCarbonError" type="real" dimensions="Time" units="kg"
+                     description="Absolute carbon conservation error over conservation interval"
+                />
+                <var name="relativeCarbonError" type="real" dimensions="Time" units="1"
```

Comment:
@darincomeau can you check the units format here? I think your recent PR #5027 changes `units="1"` to `units=""`, correct? Should change here and line 223.

Reply:
No, this is correct.  `1` is the correct unit for something that is dimensionless but physical like this.

Reply:
We never want `units=""`.  We would get rid of `units` as an attribute for something that simply doesn't have units.

Reply:
Yes, agree with @xylar - `1` is the correct units here. Thanks for chiming in!

Reply:
Great, thanks!